x/ccv/provider/keeper/keeper.go

```diff
@@ -826,3 +828,44 @@ func (k Keeper) DeleteConsumerClientId(ctx sdk.Context, chainID string) {
 	store := ctx.KVStore(k.storeKey)
 	store.Delete(types.ChainToClientKey(chainID))
 }
+
+// ------
+
+// SetInitTimeoutTimestamp sets the init timeout timestamp for the given chain ID
+func (k Keeper) SetInitTimeoutTimestamp(ctx sdk.Context, chainID string, ts uint64) {
+	store := ctx.KVStore(k.storeKey)
+	tsBytes := make([]byte, 8)
+	binary.BigEndian.PutUint64(tsBytes, ts)
+	store.Set(types.InitTimeoutTimestampKey(chainID), tsBytes)
+}
+
+// GetInitTimeoutTimestamp returns the init timeout timestamp for the given chain ID.
+func (k Keeper) GetInitTimeoutTimestamp(ctx sdk.Context, chainID string) (uint64, bool) {
```

Comment:
Some of these methods are purely test helpers, right? Might be useful to mark which ones